src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -72,31 +64,15 @@ const ProfileSection: React.FC<Props> = ({
 };
 
 const Wrapper = styled.div`
-  width: 100%;
   background-color: ${props => props.theme.colors.bg[3]};
-  margin-bottom: 64px;
-`;
-
-const Item = styled.div`
-  width: 100%;
-  &:not(:last-child) {
-    margin-bottom: 40px;
-  }
-`;
-
-const Header = styled(Text)`
-  width: 225px;
-  display: inline-block;
-`;
-
-const Content = styled.div`
-  display: inline-block;
 `;
 
 const StyledIcon = styled(Icon)`
-  display: inline;
-  float: right;
   cursor: pointer;
+
+  &:hover {
+    color: ${props => props.theme.colors.text.strong};
```

Comment:
If you are using colors directly, I believe you can import it above with styled and then pass it in like:
`${({colors}) => colors.blahblah}`